Practicals/prac_06/guitar.py

```diff
@@ -1,24 +1,32 @@
 """
 CP1404 Prac 6 - Guitar
-
+Class for guitar
 
 Estimate: 30 minutes
-Actual:
+Actual (guitar test): 34 minutes
+Actual (guitars) = 2 hrs 30 minutes
 """
 
+current_year = 2022
+vintage_criteria = 50
```

Comment:
vintage_criteria it will be changed or is it just a rule like a constant in this scenario?